pkg/resources/templates/config/state_management.go

```diff
@@ -17,6 +17,10 @@ var StateManagementTemplate = `<?xml version="1.0" encoding="UTF-8" standalone="
         <property name="Root Node">{{ .ZookeeperPath }}</property>
         <property name="Session Timeout">10 seconds</property>
         <property name="Access Control">Open</property>
+    </cluster-provider>    
+    <cluster-provider>
+        <id>kubernetes-provider</id>
+        <class>org.apache.nifi.kubernetes.state.provider.KubernetesConfigMapStateProvider</class>
```

Comment:
Will the presence of this class configuration cause issues with NiFi < 2.0.0 startup since the class doesn't exist in those versions?

Reply:
I will check.

Reply:
Tested with 1.24 and 1.21. No error.

Reply:
Tested with 1.24, 1.21 and 1.18. No error.